### DISCUSSION TITLE
Leistungsbegrenzung kann ich nicht finden

### DISCUSSION THREAD

**Discussion by Kajostro** [tbnobody/OpenDTU#932] [Q&A]
Hallo Zusammen.
Ich bitte um Gnade, aber ich bin neu hier und kenne mich nicht wirklich aus.
Ich habe mich eben mit meiner DTU angemeldet und den Wechselrichter hinzugefügt.
Ich bin mir jetzt nicht sicher, ob alles richtig angemeldet ist, da im Moment kein Strom von der PV runter kommt und ich es im Live-View nicht ablesen kann.
Was mir aber hier fehlt, ist das Leistungsbegrenzungsmenü.
Kann mir bitte jemand helfen?!
![keine Leistungsbegrenzung](https://github.com/tbnobody/OpenDTU/assets/134002590/da9cb789-a0a6-446c-8ff7-27c850156036)

DANKE!

**Comment by Omega13x** [comment]
Hast du denn alle Daten eingegeben? Pv-Module usw. Es sieht so aus als hättest du Den WR nicht fertig konfiguriert.

**Reply by Kajostro** [reply]
Der WR sollte fertig konfiguriert sein.?
![WR-Daten](https://github.com/tbnobody/OpenDTU/assets/134002590/8dc90bea-a995-445c-8221-8b001f496193)

**Comment by Omega13x** [comment]
Ist der NRF verbunden?
![image](https://github.com/tbnobody/OpenDTU/assets/123419851/728f161d-4bac-47e5-8fe8-1a94035d57f7)
Welchen Wechselrichter hast du eigentlich? Genauer Typ.

**Reply by Kajostro** [reply]
NFR sollte verbunden sein.
Ich hab den Hoymiles HM-1500.
![Systeminformationen](https://github.com/tbnobody/OpenDTU/assets/134002590/b9f2fd63-7908-487e-aa00-9081a1eec4af)

**Comment by Omega13x** [comment]
DTU mal neu gestartet? Mal ne älter Version testen. Ich habe im Moment die "v23.5.3" am laufen. Die neuste funktioniert bei mir auch nicht.

**Reply by Kajostro** [reply]
Bei mir steht auch die "v23.5.3" dran. 😔

**Comment by Omega13x** [comment]
Hast du schon mal bei Discord "Hoymiles research" rein geschaut? Da gibt es viele kluge Köpfe, die dir bestimmt weiter helfen können. Ich kann dir leider nicht weiter helfen.

**Reply by Kajostro** [reply]
Ok, schade.
Trotzdem danke!

**Comment by tbnobody** [comment]
Mach doch mal bitte einen Screenshot von Einstellungen --> Wechselrichter

**Reply by Kajostro** [reply]
Siehe oben?

**Reply by tbnobody** [reply]
Nicht von den Details... von der Liste... du darfst auch gerne die rechten paar stellen der seriennummer schwärzen. aber die linken 4-5 stellen sowie der erkannte typ wäre schon interessant.

**Reply by Kajostro** [reply]
was sagen die Zahlen aus?
![WR Liste](https://github.com/tbnobody/OpenDTU/assets/134002590/c972ca94-91f0-4ab5-a973-c02099d6bd78)

**Reply by tbnobody** [reply]
Interessanter ist eher das bei typ unbekannt steht. d.h.. du hast dich irgendwo bei der seriennummer vertippt. 1161 identifiziert schon einen HM-1500. Deine Seriennummer ist 13 stellen lang? Hast du ggf. leerzeichen mit eingetippt?

**Reply by Kajostro** [reply]
Ahhh. Krass!
Vielen Dank! Habe eine Zahl doppelt eingegeben 🙆
Besten Dank für die Hilfe!!

**Comment by h3tz** [comment]
oder so: 
<img width="470" alt="image" src="https://github.com/tbnobody/OpenDTU/assets/6421402/91517bcb-d7ce-49dc-9257-4e9b29251f81">
